package-lock.json

```diff
@@ -332,9 +332,9 @@
       "dev": true
     },
     "agent-base": {
-      "version": "4.2.0",
-      "resolved": "https://registry.npmjs.org/agent-base/-/agent-base-4.2.0.tgz",
-      "integrity": "sha512-c+R/U5X+2zz2+UCrCFv6odQzJdoqI+YecuhnAJLa1zYaMc13zPfwMwZrr91Pd1DYNo/yPRbiM4WVf9whgwFsIg==",
+      "version": "4.2.1",
+      "resolved": "https://registry.npmjs.org/agent-base/-/agent-base-4.2.1.tgz",
+      "integrity": "sha512-JVwXMr9nHYTUXsBFKUqhJwvlcYU/blreOEUkhNR2eXZIvwd+c+o5V4MgDPKWnMS/56awN3TRzIP+KoPn+roQtg==",
```

Comment:
please revert changes to this file, they shouldn't be necessary for this PR.

Reply:
More specifically, this was as a result of running `npm audit fix`

Reply:
okay, makes sense. I do run these fixes occasionally as well. I'm fine with leaving it this way for this PR, but please just create separate PRs for separate change sets in the future. This makes coordination of PRs easier and also lowers the risk of getting a merge conflict in the package-lock.json file if it takes a bit longer to integrate a feature PR like this :)

Reply:
Understood 👍